.github/workflows/test-emboss-6.6.0.yml

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
It looks great. It really does.

StaPH-B/docker-builds updated so that these files are no longer needed.